dts/bindings/rtc/rtc.yaml

```diff
@@ -36,9 +36,10 @@ properties:
       type: stringlist
       category: optional
       description: names of required interrupts
+
     prescaler:
       type: int
       category: required
-      description: Prescaler value denotes number of RTC ticks in 1 second
+      description: RTC frequency equals clock-frequency divided by the prescaler value
```

Comment:
just to clarify, once more; "prescaler value" or "2^prescaler_value")´?

Reply:
in case of rtc it's prescaler value. 2^prescaler is in TIMER and there different bindings are used.

Reply:
ok